code/services/cmc-api/src/main.rs

```diff
@@ -0,0 +1,57 @@
+use axum::{http::StatusCode, routing::get, Router};
+use clap::Parser;
+use std::net::SocketAddr;
+
+#[derive(clap::Parser)]
+pub struct Cli {
+	/// The address to bind the server to.
+	address: SocketAddr,
+}
+
+#[tokio::main]
+async fn main() {
+	tracing_subscriber::fmt::init();
+
+	let cli = Cli::parse();
+
+	let app = build_router();
+
+	let addr = cli.address;
+	tracing::info!("listening on {}", addr);
+	axum::Server::bind(&addr).serve(app.into_make_service()).await.unwrap();
+}
+
+async fn healthcheck() -> &'static str {
+	"Hello, World!"
+}
+
+const BILLION: u128 = 1_000_000_000;
+
+#[axum_macros::debug_handler]
+async fn total_supply() -> Result<String, StatusCode> {
+	Ok((BILLION * 10).to_string())
+}
+
+#[axum_macros::debug_handler]
+async fn circulating_supply() -> Result<String, StatusCode> {
+	Ok((BILLION + BILLION / 2).to_string())
```

Comment:
How do we arrive at this number?

Reply:
Currently the crowdloaners :) Will update this as we start vestings for investors/make it dynamic.